assets/js/theme/common/carousel.js

```diff
@@ -2,20 +2,44 @@ import 'slick-carousel';
 
 export default function () {
     const $carousel = $('[data-slick]');
-    if ($carousel.length) {
-        const multipleSlides = $carousel[0].childElementCount > 1;
-        $carousel.slick({ dots: multipleSlides });
-    }
+
+    if ($carousel.length === 0) return;
+
+    const multipleSlides = $carousel[0].childElementCount > 1;
+    $carousel.slick({ dots: multipleSlides });
```

Comment:
🍹 You use `multipleSlides` variable only in one place, so, I think you could combine these two rows.

```
$carousel.slick({ dots: $carousel[0].childElementCount > 1 });
```